pkg/localmetrics/localmetrics.go

```diff
@@ -155,6 +155,9 @@ func UpdateAccountCRMetrics(accountList *awsv1alpha1.AccountList) {
 	MetricTotalAccountCRsFailed.With(prometheus.Labels{"name": "aws-account-operator"}).Set(float64(failedAccountCount))
 	MetricTotalAccountCRsReady.With(prometheus.Labels{"name": "aws-account-operator"}).Set(float64(readyAccountCount))
 	MetricTotalAccountReuseFailed.With(prometheus.Labels{"name": "aws-account-operator"}).Set(float64(reuseAccountFailedCount))
+
+	// Cleanup stale IDs
+	MetricTotalAccountReusedAvailable.Reset()
```

Comment:
Due to the data we're tracking here, we do want these old IDs to stay around until we are done with them

Reply:
Oh, sorry about that. Let me remove this change. Since there is not any metrics related change, I will rename the PR title to `fix some typos`

Reply:
:+1: